e2e/app/page/workspace-data-page.ts

```diff
@@ -220,4 +220,13 @@ export default class WorkspaceDataPage extends WorkspaceBase {
     await this.waitForLoad();
     return cloneName;
   }
+
+  // extract the workspace-namesapce from the url to be used to load workspace in the workspace-admin page
+  async extractWorkspaceNamespace(): Promise<string> {
+    const href = await page.evaluate(() => location.href);
+    console.log(`href: ${href}`);
```

Comment:
`page` is a global object. It should be `this.page`. Please check and make sure no other pages are using global `page` object.
`console.log` is not needed.